table/block_based_filter_block.h

```diff
@@ -99,17 +102,13 @@ class BlockBasedFilterBlockReader : public FilterBlockReader {
 
  private:
   const FilterPolicy* policy_;
-  const SliceTransform* prefix_extractor_;
+  ROCKSDB_FIELD_UNUSED const SliceTransform* prefix_extractor_;
```

Comment:
what is this for?

Reply:
This will cause warning from compiler and will terminate compiling with -Werror.